src/tools/admin/conseillers/import-recrutes.js

```diff
@@ -126,6 +133,10 @@ execute(__filename, async ({ feathers, app, db, logger, exit, Sentry }) => {
             logger.error(`Mise en relation introuvable pour la structure avec l'idPG '${structureId}'`);
             Sentry.captureException(`Mise en relation introuvable pour la structure avec l'idPG '${structureId}'`);
             errors++;
+          } else if (dateRupture && (dateRupture > miseEnRelation.dateRecrutement)) {
+            // eslint-disable-next-line max-len
+            logger.error(`Un conseiller avec l'id: ${idPGConseiller} a une date de Rupture ${formatDate(dateRupture)} supérieure à la date de recrutement ${formatDate(miseEnRelation.dateRecrutement)}`);
```

Comment:
ok comme ca reste prévisionnelle, faudra bien noté ces erreurs remontées pour pas trop que ca bloque les import coop